<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="easySetting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>简单连点设置</string>
    </property>
    <widget class="QLabel" name="clickTypeText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>点击类型:</string>
     </property>
    </widget>
    <widget class="QLabel" name="clickTimeText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>间隔时间:</string>
     </property>
    </widget>
    <widget class="QLabel" name="quickStartText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>快捷热键:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="clickType">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>鼠标左键</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>鼠标滑轮</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>鼠标右键</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="clickTime">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="quickStart">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>F1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F12</string>
      </property>
     </item>
    </widget>
    <widget class="QTextBrowser" name="howToUse">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600;&quot;&gt;使用说明&lt;/span&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;1. 将鼠标移动到需要连点的地方；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;2. 然后按下 &lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; color:#ff0000;&quot;&gt;Ctrl + F8&lt;/span&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt; 即可开始鼠标连点操作，热键 &lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; color:#ff0000;&quot;&gt;F8&lt;/span&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt; 可更改。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="normalSetting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>普通连点设置</string>
    </property>
    <widget class="QCheckBox" name="randomClick">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>启动随机间隔时间</string>
     </property>
    </widget>
    <widget class="QComboBox" name="randomClickTime">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10.0</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="randomClickTimeText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>最大间隔时间</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="advancedSetting">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>高级连点设置</string>
    </property>
    <widget class="QPushButton" name="newCreate">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="oldRecord">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="howToUse_adv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:7pt; font-weight:600;&quot;&gt;使用说明：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:7pt;&quot;&gt;1. 点击“新建”按钮打开“录制工具栏”；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:7pt;&quot;&gt;2. 录制鼠标点击事件；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:7pt;&quot;&gt;3. 点击“列表”按钮打开“记录栏”；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:7pt;&quot;&gt;4. 启动对应点击事件。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:7pt; font-weight:600;&quot;&gt;Tips：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:7pt;&quot;&gt;使用快捷键可以更方便哦！(●'◡'●)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="newCreateText">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>新建</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="oldRecordText">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>列表</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
